src/applications/check-in/day-of/README.md

```diff
@@ -142,7 +142,7 @@ Though we have the HOC, its now considered best practice to query redux using th
 - `check_in_experience_travel_reimbursement`: Enables travel reimbursement workflow for day-of check-in.
 - `check_in_experience_travel_logic`: Enables logic that allows the veteran to submit a travel claim to more than one appointment in a day if they are at separate facilities.
 - `check_in_experience_browser_monitoring`: Enables browser monitoring for check-in applications.
-- `check_in_experience_unified_landing_page`: Enabled the unified check in landing page
+- `check_in_experience_upcoming_appointments_enabled`: Enables the upcoming appointments block.
```

Comment:
I think we can move this list and the one from pre-check-in to the root level readme?

Reply:
oh that makes sense to me too. Good idea!